### ISSUE TITLE
Batch normalization is being applied before the activation (opposite of Keras!)

### ISSUE THREAD

**Issue by Astlaan** [keras-team/autokeras#1175]
From the DenseBlock class:
```
        for i in range(num_layers):
            units = hp.Choice(
                'units_{i}'.format(i=i),
                [16, 32, 64, 128, 256, 512, 1024],
                default=32)
            output_node = layers.Dense(units)(output_node)
            if use_batchnorm:
                output_node = layers.BatchNormalization()(output_node)
            output_node = layers.ReLU()(output_node)
            if dropout_rate > 0:
                output_node = layers.Dropout(dropout_rate)(output_node)
        return output_node
```
Whereas in Keras the default usage is:
output = Dense(...)(input)
output = BatchNormalization(output)

Check:
https://github.com/keras-team/keras/issues/1802

**Comment by haifeng-jin** [comment]
Thank you for the issue! We may make it more flexible to have different orders of the layers as a hyperparameter. I think people are doing it in various ways.
